### PR TITLE
change and extend status layout

### PR DESCRIPTION
**Pull request by rmsilva**
Extend layout in a more graphic way similiar to
https://linaro.github.io/msm/ but for Laptops status
feature support, with a dedicated page with more
information for x13s and t14s lenovo laptops.

### CODE REVIEW

**Comment by obbardc** [comment]
Can you share a rendered link/screenshots in the body? Also needs rebasing.

**Comment by Stefan-Schmidt** [comment]
image
@obbardc

File: _laptop/thinkpad_x13s.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+---
+name: ThinkpadX13s
```
Comment (Stefan-Schmidt) [inline _laptop/thinkpad_x13s.md]: Having an extra space here for Thinkpad X13s (same for T14) would look better.

File: _laptop/thinkpad_x13s.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+layout: laptop
+status-audio: N/A
+status-audio-displayportaudio: N
+status-audio-mic: 6.5
+status-audio-mic-comment: "
+    Some capture issues
+"
+status-audio-speakers: 6.5
+status-audio-speakers-comment: "
```
Comment (Stefan-Schmidt) [inline _laptop/thinkpad_x13s.md:11]: I like how the comment has double use on the inividual davice page and as tooltip on the main matrix. Nice! :-)

**Comment by obbardc** [comment]
> image @obbardc

I'd rather see the row/columns transposed.

File: _laptop/thinkpad_x13s.md (hunk 1 above)
Comment (rmsilva) [reply _laptop/thinkpad_x13s.md]: Added, need to change also the wrapping stuff but done.

File: _laptop/thinkpad_x13s.md (hunk 2 above)
Comment (rmsilva) [reply _laptop/thinkpad_x13s.md:11]: yeah, not to have the credits, the framework works like that by default. ehehe

**Comment by rmsilva** [comment]
> > image @obbardc
> 
> I'd rather see the row/columns transposed.

The idea here is to be visual and informational coherent with https://linaro.github.io/msm/,
to have the same heatmap layout, and for a vertical info you just click on the respective laptop and you have a more detailed vertical information:

image

File: _laptop/thinkpad_x13s.md (hunk 2 above)
Comment (Stefan-Schmidt) [reply _laptop/thinkpad_x13s.md:11]: Fair enough, keep the credits though :-)

**Comment by Stefan-Schmidt** [comment]
> > image @obbardc
> 
> I'd rather see the row/columns transposed.

Given our screen estate nowadays is very much 16:9, or at least skewed towards landscape ratio, I actually prefer the horizontal layout. If we end up with enough devices we might end up in square ;-)

**Review by Stefan-Schmidt — APPROVED** [review]

**Comment by obbardc** [comment]
> > > image @obbardc
> > 
> > 
> > I'd rather see the row/columns transposed.
> 
> Given our screen estate nowadays is very much 16:9, or at least skewed towards landscape ratio, I actually prefer the horizontal layout. If we end up with enough devices we might end up in square ;-)

Sure. We can always modify later if needed :-)

**Review by obbardc — APPROVED** [review]

**Comment by rmsilva** [comment]
@obbardc @Stefan-Schmidt thanks for the reviews, merged

**Pull request merged**